test/forge/MixedSwapCallParameters.t.sol

```diff
@@ -54,6 +57,53 @@ contract MixedSwapCallParameters is Test, Interop, DeployRouter {
         assertEq(nft.balanceOf(RECIPIENT), 1);
     }
 
+    function testMixedWETHForLooksRareNFTWithPermit() public {
+        MethodParameters memory params = readFixture(json, "._PERMIT_AND_WETH_FOR_1_LOOKSRARE_NFT");
+
+        ERC721 nft = ERC721(0x5180db8F5c931aaE63c74266b211F580155ecac8);
```

Comment:
Maybe pull this address out somewhere as a magic constant? e.g. `CRYPTO_COVEN_ADDRESS` at the top?

Reply:
Also all the tests start with the same line so it could be in the setup function